templates/rss.ftl

```diff
@@ -1,11 +1,13 @@
+<#setting locale="en_US">
+<#setting time_zone="GMT">
+<#setting datetime_format="EEE, d MMM yyyy HH:mm:ss z">
```

Comment:
Not sure it should include "HH:mm:ss Z". Looking at https://www.w3schools.com/xml/rss_tag_pubdate_item.asp it doesn't. Also, we don't have HH:mm:ss available, so will always be 00:00:00.

Reply:
I'm quite sure: pubDate in [RSS 2.0 Spec](http://www.rssboard.org/rss-specification#ltpubdategtSubelementOfLtitemgt): `pubDate` is RFC822 -> [RFC 822 Spec](https://www.ietf.org/rfc/rfc822.txt)

*But* I think it's OK without time and time zone. Good RSS parsers, which need to handle many spec-violations in the wild, should be fine with it 😄